Minio/Helper/OperationsHelper.cs

```diff
@@ -0,0 +1,32 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Minio
+{
+    public class OperationsUtil
+    {
+        internal static bool IsSupportedHeader(string hdr, IEqualityComparer<string> comparer = null)
+        {
+            comparer = comparer ?? StringComparer.OrdinalIgnoreCase;
+            var supportedHeaders = ImmutableArray.Create<string>(new string[]{ "cache-control", "content-encoding", "content-type", "x-amz-acl", "content-disposition" });
```

Comment:
there seems to be 2 other definitions for supported headers - why does this need to be available as a public class. Can you unify them? This is internal to sdk

Reply:
Yes. I was eventually going to replace supportedHeaders. I will do that in this PR itself by using this function.